nucliadb/nucliadb/tests/integration/search/test_search.py

```diff
@@ -1347,11 +1347,11 @@ async def test_search_two_logic_shards(
     # Check that search returns the same results
     resp1 = await nucliadb_reader.post(
         f"/kb/{kbid1}/search",
-        json=dict(query="dummy", vector=V1, min_score=-1, with_duplicates=True),
+        json=dict(query="dummy", vector=V1, min_score=-1),
```

Comment:
IIRC the `with_duplicate` change was an unrelated fix

Reply:
yes, but AFAIK this other fix was included in the same patch. not sure how easy it is to cherry pick